tests/WebAppCallsWebApiCallsGraph/TodoListService/Controllers/Callback.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
You might want to rename this file to CallbackController.cs

Reply:
done.